### PR TITLE
Kotlin cleanup of NoteEditorTest.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin cleanup of NoteEditorTest.kt

### CODE REVIEW

**Review by david-allison — APPROVED** [review]

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt [hunk 1]
```diff
@@ -30,24 +30,22 @@ import org.junit.Before
...
     )
 
-    @KotlinCleanup("simplify property getter with apply and direct return")
     private val noteEditorIntent: Intent
         get() {
-            val intent = Intent(targetContext, NoteEditor::class.java)
-            intent.component = ComponentName(targetContext, NoteEditor::class.java)
-            intent.putExtra(NoteEditor.EXTRA_CALLER, NoteEditor.CALLER_DECKPICKER)
-            return intent
+            return Intent(targetContext, NoteEditor::class.java).also {
```
Comment (lukstbit) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:45]: `apply {}`(as requested by the cleanup) is a better solution here as it avoids the need of using `it`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt (hunk 1 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:45]: i not understand what i have to do ?

Comment (lukstbit) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:45]: The cleanup that you deleted said:

> @KotlinCleanup("simplify property getter with **_apply_** and direct return")

For some reason you decided to use `also` which is more verbose.

Comment (TimSta82) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:45]: replace `also {}` with `apply {}`

within 
apply {
   component = ...
   putExtra(...)
}

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt [hunk 2]
```diff
@@ -63,8 +61,7 @@ abstract class NoteEditorTest protected constructor() {
         }
     }
 
-    @KotlinCleanup("Simplify property getter: too many lists created")
-    protected val invalidSdksImpl: List<Int>
+    private val invalidSdksImpl: List<Int>
```
Comment (TimSta82) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:64]: have you simplified the getter? 
I think something like this is meant:
`
val invalid: Int = Build.VERSION_CODES.N_MR1
val integers: ArrayList<Int> = arrayListOf(invalid)
integers.addAll(invalidSdks!!)
return integers
`

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt (hunk 1 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt]: ok

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt (hunk 2 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/NoteEditorTest.kt:64]: No getter is fine there is no need to simplify getter

**Review by lukstbit — APPROVED** [review]

**Pull request merged**